setup.py

```diff
@@ -14,16 +14,6 @@
     os.system("python setup.py sdist upload")
     sys.exit()
 
-if sys.argv[-1] == 'speedups':
-    try:
-        __import__('pip')
-    except ImportError:
-        print('Pip required.')
-        sys.exit(1)
-
-    os.system('pip install ujson')
```

Comment:
I think it removed a feature (speed). We can try to import ujson if it is available.

```
try:
    import ujson as json
except ImportError:
    import json